versioned_docs/version-1.2/how_to_guides/multiarch.mdx

```diff
@@ -0,0 +1,135 @@
+✓ Exists    application/vnd.unknown.config.v1+json                                                                                       37/37  B 100.00%     0s
+  └─ sha256:d6f56bc20064075ce319ac2e6fcef5de9ea21773b0a8a4398c4405222971f9c0
+✓ Uploaded  application/vnd.oci.image.manifest.v1+json                                                                                 581/581  B 100.00%     0s
+  └─ sha256:302df4334cffe5132b6b81c06742be8ab3345bff3ce922662bc8674e79113bfd
+Pushed [registry] localhost:15000/oras:linux
+Tagged arm64
+ArtifactType: application/vnd.oci.image.config.v1+json
+Digest: sha256:302df4334cffe5132b6b81c06742be8ab3345bff3ce922662bc8674e79113bfd
+```
+
+Next, we push a Linux AMD64 executable:
+```shell
+% oras push --plain-http localhost:15000/oras:linux,amd64  --platform linux/amd64 --artifact-type 'application/vnd.oci.image.config.v1+json'  bin/linux/amd64/oras:application/octet-stream
+✓ Uploaded  bin/linux/amd64/oras                                                                                                         11/11 MB 100.00%     0s
+  └─ sha256:4640fefdb4e22eb1308e40d5fac9fd9d16e17dfbc20202bb3ed42033d01d823e
+✓ Exists    application/vnd.unknown.config.v1+json                                                                                       37/37  B 100.00%     0s
+  └─ sha256:9d99a75171aea000c711b34c0e5e3f28d3d537dd99d110eafbfbc2bd8e52c2bf
+✓ Uploaded  application/vnd.oci.image.manifest.v1+json                                                                                 581/581  B 100.00%     0s
+  └─ sha256:e5b0ffe309a91893d4423f09e82410364607fd749ea2f93dd1c286d718d9c1e4
+Pushed [registry] localhost:15000/oras:linux
+Tagged amd64
+ArtifactType: application/vnd.oci.image.config.v1+json
+Digest: sha256:e5b0ffe309a91893d4423f09e82410364607fd749ea2f93dd1c286d718d9c1e4
+```
+This example pushes the executable for two architectures, but you could also push artifacts with a different operating system or other platform attribute.
+
+## Create a multiple architecture manifest
+
+After you have pushed all your platform specific artifacts, create a manifest referencing them:
+```shell
+% oras manifest index create localhost:15000/oras:v1 sha256:302df4334cffe5132b6b81c06742be8ab3345bff3ce922662bc8674e79113bfd sha256:e5b0ffe309a91893d4423f09e82410364607fd749ea2f93dd1c286d718d9c1e4
+Fetching  sha256:302df4334cffe5132b6b81c06742be8ab3345bff3ce922662bc8674e79113bfd
+Fetched   sha256:302df4334cffe5132b6b81c06742be8ab3345bff3ce922662bc8674e79113bfd
+Fetching  sha256:e5b0ffe309a91893d4423f09e82410364607fd749ea2f93dd1c286d718d9c1e4
+Fetched   sha256:e5b0ffe309a91893d4423f09e82410364607fd749ea2f93dd1c286d718d9c1e4
+Packed    0fd4ac889c6f application/vnd.oci.image.index.v1+json
+Pushed    [registry] localhost:15000/oras:v1
+Digest: sha256:0fd4ac889c6f1f3e4fbb1c88b0cc18b97176bf7fd0841ac9c5264d20dd54e2d8
+```
+
+## Download your platform specific artifact
+
+Create a download directory:
+```shell
```

Comment:
No need to create a download directory, file store will create all the required folder on-demand

Reply:
Done